src/DepthImageToLaserScanROS.cpp

```diff
@@ -92,7 +91,7 @@ void DepthImageToLaserScanROS::infoCb(sensor_msgs::msg::CameraInfo::SharedPtr in
 void DepthImageToLaserScanROS::depthCb(const sensor_msgs::msg::Image::SharedPtr image)
 {
   if (nullptr == cam_info_) {
-    ROS_ERROR("No camera info, skipping point cloud squash\n");
+    RCUTILS_LOG_ERROR("No camera info, skipping point cloud squash\n");
```

Comment:
you probably don't want to add the newline manually anymore (and below)

Reply:
Good call, I'll fix that.